components/automate-ui/src/app/entities/data-bags/data-bags.effects.ts

```diff
@@ -64,7 +64,7 @@ export class DataBagsEffects {
     map(({ payload: { databag: dataBag } }: CreateDataBagSuccess) => {
       return new CreateNotification({
         type: Type.info,
-        message: `Successfully Created Data Bag ${dataBag.name}.`
+        message: `Successfully created data bag - ${dataBag.name}.`
```

Comment:
Author note: consistency -- this new pattern is supposed to also include the hyphen.